sentences/el/homeassistant_HassTurnOn.yaml

```diff
@@ -3,7 +3,16 @@ intents:
   HassTurnOn:
     data:
       - sentences:
-          - <turn_on> <articles> <name>
-          - <articles> <name> να <turn_on>
-          - να <turn_on> <articles> <name>
-          - να <turn_on> <articles> <name>
+          - <turn_on> <articles> <n>
+          - <articles> <n> να <turn_on>
+          - να <turn_on> <articles> <n>
+          - να <turn_on> <articles> <n>
+        excludes_context:
```

Comment:
Why exclude all these domains ? This is only a PR for scene.

I guess we would never match on sensor anyway?